.github/workflows/ci.yml

```diff
@@ -0,0 +1,37 @@
+name: CI
+on: 
+  push:
+  pull_request:
+    branches:
+      - development
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: '10'
+      - run: npm ci
+      - run: npm run front-end:test
+      - run: npm run back-end:test
+  lint: 
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: '10'
+      - run: npm ci
+      - run: npm run front-end:lint
+      - run: npm run back-end:lint
+  audit:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: '10'
+      - run: npm ci
+      - run: npm audit --audit-level=moderate
```

Comment:
```suggestion
```
I would propose removing the `audit` job from the `develop` branch until such a time as the `audit` job is passing. So I'm not really proposing deleting but rather extracting this job into a feature and merging when this feature is passing–I'm suggesting establishing an effective [RGR cycle](https://www.codecademy.com/articles/tdd-red-green-refactor) as part of the pull request process. The steps would be:
1. Make a new PR that removes this block, per the suggestion.
2. Merge the new PR to `develop` so CI is green again.
3. Make a new draft PR that adds this block back to the CI configuration.
4. Commit any necessary fixes to resolve the failing CI stage (ie make the Red branch turn Green)
5. Refactor as needed to respond to peer review (ie keep the Green branch Green, while getting ready for merge). Consider requesting an org review as the final step when ready to merge–there's [information in another PR](https://github.com/bcgov/digital_marketplace/pull/188#issuecomment-974319379) that indicates an external reviewer might be able to add value to this proposed future PR.
6. Merge the working audit CI stage to become a new increment of value on the CI branch in tandem with the fixes that allow it to be a new effective guardrail moving forward (ie an already green `develop` will stay green and future PRs will now be subject to `npm audit` guardrail).

Reply:
@wenzowski - Thanks, this looks great.  The step-by-step explanation provides makes it easy to ticket the required work.